tests/backends/test_schema_parity/test_columns.yml

```diff
@@ -0,0 +1,300 @@
+db_dbauthinfo:
+  aiidauser_id:
+    data_type: integer
+    default: null
+    is_nullable: false
+  auth_params:
+    data_type: jsonb
+    default: null
+    is_nullable: false
+  dbcomputer_id:
+    data_type: integer
+    default: null
+    is_nullable: false
+  enabled:
+    data_type: boolean
+    default: null
+    is_nullable: false
+  id:
+    data_type: integer
+    default: nextval('db_dbauthinfo_id_seq'::regclass)
+    is_nullable: false
+  metadata:
+    data_type: jsonb
+    default: null
+    is_nullable: false
+db_dbcomment:
+  content:
+    data_type: text
+    default: null
+    is_nullable: false
+  ctime:
+    data_type: timestamp with time zone
+    default: null
+    is_nullable: false
+  dbnode_id:
+    data_type: integer
+    default: null
+    is_nullable: false
+  id:
+    data_type: integer
+    default: nextval('db_dbcomment_id_seq'::regclass)
+    is_nullable: false
+  mtime:
+    data_type: timestamp with time zone
+    default: null
+    is_nullable: false
+  user_id:
+    data_type: integer
+    default: null
+    is_nullable: false
+  uuid:
+    data_type: uuid
+    default: null
+    is_nullable: false
+db_dbcomputer:
+  description:
+    data_type: text
+    default: null
+    is_nullable: false
+  hostname:
+    data_type: character varying
+    default: null
+    is_nullable: false
+    max_length: 255
+  id:
+    data_type: integer
+    default: nextval('db_dbcomputer_id_seq'::regclass)
+    is_nullable: false
+  label:
+    data_type: character varying
+    default: null
+    is_nullable: false
+    max_length: 255
+  metadata:
+    data_type: jsonb
+    default: null
+    is_nullable: false
+  scheduler_type:
+    data_type: character varying
+    default: null
+    is_nullable: false
+    max_length: 255
+  transport_type:
+    data_type: character varying
+    default: null
+    is_nullable: false
+    max_length: 255
+  uuid:
+    data_type: uuid
+    default: null
+    is_nullable: false
+db_dbgroup:
+  description:
+    data_type: text
+    default: null
```

Comment:
Are these values correct? This says that `DbGroup.description`'s default is `null` but in the models it actually has `default=''`. I was wondering what the hell was going on with the defaults where for many columns you are now defining `default = ''` but not all of them. I had already added quite a few review comments at columns that don't specify a default, asking whether they should. What was the general thinking as to which fields need a default and which don't?

In any case, if the tests pass with this reference file, that seems a problem no? All defaults seem to be null here even though most models actually define one.

Reply:
Note, everything in these regression files are server side data; `null` here simply means that no default has been set. By contrast, `default=''` on the model is a client side default